README.md

```diff
@@ -175,9 +217,26 @@ Registering a new Salesforce Push Topic to listen for. Part of the salesforce co
     pushTopic.NotifyForOperationDelete = true;
     pushTopic.NotifyForFields = 'All';
     insert pushTopic;
+    ```
     
-5. Set up environment variables for salesforce topic polling user, INCLUDING security token - process.env.SF_TOPIC_USERNAME, process.env.SF_TOPIC_PASSWORD
+1. Set up environment variables for salesforce topic polling user, INCLUDING security token - process.env.SF_TOPIC_USERNAME, process.env.SF_TOPIC_PASSWORD
 
-6. Register for this notification in node-salesforce
+1. Register for this notification in node-salesforce
 
 More info: http://wiki.developerforce.com/page/Getting_Started_with_the_Force.com_Streaming_API
+
+## Tests
+
+All the tests are in the "test/" directory. The cloud app is using mocha as the test runner. 
+
+### Unit tests
```

Comment:
I needed to run `grunt serve` to get the tests to run so could we add this to the README and that mongod needs to be running locally too